Sources/Controllers/TargetMenu/PointEditing/OAEditorViewController.h

```diff
@@ -0,0 +1,44 @@
+//
+//  OAEditorViewController.h
+//  OsmAnd
+//
+//  Created by Skalii on 11.10.2023.
+//  Copyright © 2023 OsmAnd. All rights reserved.
+//
+
+#import "OABaseNavbarSubviewViewController.h"
+
+@class OAGPXAppearanceCollection, OATextInputFloatingCell;
+
+@protocol OAEditorDelegate <NSObject>
+
+- (void)onEditorAdded:(NSString *)name
```

Comment:
Bad name.
It should be called by OAEditorViewController only and when item is already added.